node/src/index-send-eth-to-payid.js

```diff
@@ -0,0 +1,143 @@
+const axios = require('axios')
```

Comment:
Can we just drop this into pay-id-demo? 

If you look at the XRP demo we generally just shove a bunch of functionality after each other. I think this makes it easier to find examples by a single high level ordering.

Reply:
I think the ETH stuff should be separated out for ease of finding